README.md

```diff
@@ -363,14 +380,19 @@ browser warns the user.
 
 <img src="images/DuckDuckGo_Browser_Redirect_Warning.png" width=300px/>
 
-#### Save Browsers
+
+#### Safe Browsers
 The following browsers generate the intended 
 user experience:
 
 1. **Chrome** (especially Chrome Custom Tabs)
 2. **Firefox** (also supports Custom Tabs)
 3. **Opera** 
 
+JJH: Is this section only talking about custom url schemes?
+
+JJH: I'm interested to know what browsers were considered here - https://www.netmarketshare.com/browser-market-share.aspx?options=%7B%22filter%22%3A%7B%22%24and%22%3A%5B%7B%22deviceType%22%3A%7B%22%24in%22%3A%5B%22Mobile%22%5D%7D%7D%2C%7B%22platform%22%3A%7B%22%24in%22%3A%5B%22Android%22%5D%7D%7D%5D%7D%2C%22dateLabel%22%3A%22Custom%22%2C%22attributes%22%3A%22share%22%2C%22group%22%3A%22browser%22%2C%22sort%22%3A%7B%22share%22%3A-1%7D%2C%22id%22%3A%22browsersDesktop%22%2C%22dateInterval%22%3A%22Monthly%22%2C%22dateStart%22%3A%222020-06%22%2C%22dateEnd%22%3A%222020-06%22%2C%22segments%22%3A%22-1000%22%2C%22hiddenSeries%22%3A%7B%7D%7D suggests we should probably explicitly comment on Samsung Browser.
```

Comment:
I agree with you that we should comment on the Samsung Browser but at the moment it is difficult for me to test the Samsung Browser because I could not install the APK from APKMirror.